lib/definitions.ts

```diff
@@ -0,0 +1,19 @@
+export type EmployeesData = {
+    id: number;
```

Comment:
The type of the id that will be coming from the database is going to be a string not a number

Reply:
You can use 
id: number | string